protos/external.proto

```diff
@@ -108,6 +116,40 @@ message ResumePipelineRequest {
   protos.Audience audience = 2;
 }
 
+// Notifications
+message CreateNotificationRequest {
+  protos.NotificationConfig notification = 1;
+}
+
+message UpdateNotificationRequest {
+  protos.NotificationConfig notification = 1;
+}
+
+message DeleteNotificationRequest {
+  string notification_id = 1;
+}
+
+message GetNotificationsRequest {
+  // Don't think we need anything here
+}
+
```

Comment:
Add a comment explaining what key is.

Reply:
I think this was meant for `GetNotificationsResponse`. Added comment